### PR TITLE
Setup github actions for automatic building

### PR DESCRIPTION
**Pull request by exaptis**

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  main:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name != 'pull_request' }}
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout [main]
+        with:
+          fetch-depth: 0
+      - run: npm install
+      - run: npm run build:elements
```
Comment (01sas) [inline .github/workflows/github-actions.yml]: I don't see a logic to create a Release. In general, it is ok with me to create a Release manually, but I would like an artefact (zip archive with JS and CSS) to be published by Job at least (https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts).

**Review by 01sas — CHANGES_REQUESTED** [review]
Please publish artefacts on in the main job.

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (exaptis) [reply .github/workflows/github-actions.yml]: Yep that's true, I'll work on that but I had to create a PR first to check if the git actions are triggered and running successfully :-)

Comment (exaptis) [reply .github/workflows/github-actions.yml]: @01sas fixed. The main build will now create the artifact, which will be zipped when downloaded from git by default see https://github.com/actions/upload-artifact#zipped-artifact-downloads

File: build-elements.js [hunk 2]
```diff
@@ -12,7 +12,5 @@ const concat = require('concat');
 
   await fs.ensureDir(`${targetDir}`);
   await concat(files, `${targetDir}/snapodds-sdk.js`);
-
   await fs.copyFile(`${sourceDir}/styles.css`, `${targetDir}/snapodds-sdk.css`);
-  await fs.copyFile(`${sourceDir}/index.html`, `${targetDir}/index.html`);
```
Comment (exaptis) [inline build-elements.js:17]: index.html is not working as the wrong files are referenced.

File: apps/snapodds-builder/src/app/odds-sdk.builder.spec.ts [hunk 3]
```diff
@@ -14,7 +14,6 @@ describe('OddsSdkBuilder', () => {
   });
 
   it('should assign the tvSearchResult to the sdk', () => {
-    console.log(sportEventTvSearchMock);
```
Comment (exaptis) [inline apps/snapodds-builder/src/app/odds-sdk.builder.spec.ts:17]: cleaned up leftover

**Review by 01sas — APPROVED** [review]

**Pull request merged**